### PR TITLE
Reload emailAddresses after importing members

### PR DESCRIPTION
**Pull request by amberwebb**

### CODE REVIEW

**Comment by nathanielvelez** [comment]
:+1:

File: src/persistence.coffee [hunk 1]
```diff
@@ -699,6 +701,16 @@ modifySDK = (sdk) ->
...
 
+  wrapMethod sdk, 'importMembersFromTeam', (importMembersFromTeam) ->
+    (memberIds, teamId, callback) ->
+      importMembersFromTeam.call(this, memberIds, teamId)
+      .then((result) ->
+        memberIds = result.map (member) -> member.id
+        promises.when(
+          sdk.loadMemberEmailAddresses({memberId: memberIds}).then -> result
+          sdk.loadContactEmailAddresses({memberId: memberIds}).then -> result
+        )
```
Comment (matthopson) [inline src/persistence.coffee]: You probably want to use:
```coffeescript
promises.when(
  sdk.loadMemberEmailAddresses({memberId: memberIds})
  sdk.loadContactEmailAddresses({memberId: memberIds})
).then -> result

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "teamsnap.js",
-  "version": "1.12.0",
+  "version": "1.13.0",
```
Comment (matthopson) [inline package.json]: This seems more like a `patch` update to me

File: src/persistence.coffee (hunk 1 above)
Comment (amberwebb) [inline src/persistence.coffee]: Wasn't sure about that one. Thanks.

**Comment by amberwebb** [comment]
@matthopson Please review again.

**Comment by matthopson** [comment]
:+1:

**Pull request merged**